collectors/cgroups.plugin/cgroup-name.sh.in

```diff
@@ -116,13 +133,16 @@ for CONFIG in "${NETDATA_USER_CONFIG_DIR}/cgroups-names.conf" "${NETDATA_STOCK_C
 	fi
 done
 
+if [ -z "${NAME}" ] && [ -n "${KUBERNETES_SERVICE_HOST}" ] && [ -n "${KUBERNETES_PORT_443_TCP_PORT}" ]; then
```

Comment:
KUBERNETES_PORT_443_TCP_PORT -> KUBERNETES_API_PORT

Reply:
I haven't seen KUBERNETES_API_PORT in any official docs, or even mentioned in issues. How can I trust it to be the https port?

Reply:
We'll be setting the correct RBAC in the helm chart, but you're right, it will be taken care of.

Reply:
It worked on all tests with this, so leaving it as is.

Reply:
It worked on all tests with this, so leaving it as is.